sockeye/layers.py

```diff
@@ -13,11 +13,13 @@
 
 import logging
 import math
-from typing import Dict, Optional, Tuple, Union
+from typing import Dict, Optional, Tuple, Union, List, Iterator
+from abc import ABC, abstractmethod
```

Comment:
"optimize imports" in pycharm will alphabetically sort

Reply:
done